Source: Desktop consolidation
Comment of upstream submission:
Currently the Desktop consolidation patches are re-taken without the change. When the
component will update patches need to be checked and all unneeded contents i.e. concerning
the version of auto tools need to be checked if it can be taken out.

Test of dns_sd.h should rather to be promoted to community.

--- /usr/tmp/clean/avahi-0.6.28/configure.ac	2010-10-05 00:33:15.689579806 +0100
+++ avahi-0.6.28/configure.ac	2011-01-21 09:13:07.100772101 +0000
@@ -153,6 +153,17 @@
 ACX_PTHREAD(,AC_MSG_ERROR([Missing POSIX Threads support]))
 
 #
+# Check for dns_sd.h
+#
+AC_CHECK_HEADER(dns_sd.h,
+HAVE_BONJOUR=yes
+AC_DEFINE([HAVE_BONJOUR],[],[Support for Bonjour])
+, [], [
+])
+
+AM_CONDITIONAL(HAVE_BONJOUR, [ test x"$HAVE_BONJOUR" = xyes ])
+
+#
 # Check for netlink.h
 #
 AC_CHECK_HEADER(linux/netlink.h,
